### ISSUE TITLE
Missing time series Recovery Data for all United States

### ISSUE THREAD

**Issue by davchip** [CSSEGISandData/COVID-19#1166]

**Comment by JamiroGS** [comment]
+1!

**Comment by ac7117** [comment]
+1

**Comment by decser** [comment]
+1

**Comment by isaaczhao23** [comment]
Sum of all Recoveries in the US is much less than reported in general. As of recently ~ 2020-03-20, the sum of all Recoveries in the US drops to 0. However, the [daily reports dataset](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_daily_reports/03-20-2020.csv) has the correct number of Recoveries on row 284 where Country.Region == "US" and Province.State=="US".

![image](https://user-images.githubusercontent.com/30127730/77233545-9189cd00-6b7e-11ea-90a0-0c432fdb3464.png)

![image](https://user-images.githubusercontent.com/30127730/77233528-6bfcc380-6b7e-11ea-9608-d5e748188935.png)

**Comment by davchip** [comment]
Yes but if we get regional data across China and other regions within the EU we should be able to get it here in the US. This is important because of the diversity of the US and how different each region is; this seems a bit lazy to put it all in one line item.
